solidity/contracts/converter/types/fixed-rate-pool/FixedRatePoolConverter.sol

```diff
@@ -0,0 +1,184 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+
+import "../standard-pool/StandardPoolConverter.sol";
+
+/**
+ * @dev This contract is a specialized version of the converter, which implements
+ * a constant conversion-rate (configurable by the owner of the converter).
+ */
+contract FixedRatePoolConverter is StandardPoolConverter {
+    mapping(IERC20Token => uint256) private _rate;
+
+    /**
+     * @dev initializes a new FixedRatePoolConverter instance
+     *
+     * @param  _anchor             anchor governed by the converter
+     * @param  _registry           address of a contract registry contract
+     * @param  _maxConversionFee   maximum conversion fee, represented in ppm
+     */
+    constructor(
+        IConverterAnchor _anchor,
+        IContractRegistry _registry,
+        uint32 _maxConversionFee
+    ) public StandardPoolConverter(_anchor, _registry, _maxConversionFee) {}
+
+    /**
+     * @dev returns the converter type
+     *
+     * @return see the converter types in the the main contract doc
+     */
+    function converterType() public pure override returns (uint16) {
+        return 4;
+    }
+
+    /**
+     * @dev defines a new reserve token for the converter
+     * can only be called by the owner while the converter is inactive
+     *
+     * @param _token   address of the reserve token
+     * @param _weight  reserve weight, represented in ppm, 1-1000000
+     */
+    function addReserve(IERC20Token _token, uint32 _weight) public override {
+        super.addReserve(_token, _weight);
+        _rate[_token] = 1;
```

Comment:
no need to override the function and actually better that rates will stay 0:0 until the owner sets them, so that the rate isn't 1:1 automatically and swaps occur unintentionally.

Reply:
Fixed.